catalog/task/get-ocp-version/0.1/README.md

```diff
@@ -0,0 +1,9 @@
+# get-ocp-version
+
+Tekton task to collect OCP version tag from FBC fragment using `skopeo inspect`.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | A FBC container Image | No | - |
```

Comment:
the parameter name and description don't really line up for me. Is it the entire snapshot or is it expecting just an fbc container image?

Reply:
I am waiting for Leandro's response to make sure what should be the right param for this task, For now, I tested it using this [image](quay.io/hacbs-release-tests/managed-release-team-tenant/sample-fbc-application/sample-fbc-component@sha256:3bab7aa91d6e62c6e6f8cd6e701b78124d477eab3d8d1e1d3bce59c81204b1d7) which contains the version we are looking for. I will update the Description and Name accordingly.